examples/compareLambdaTypes.py

```diff
@@ -25,30 +27,31 @@
 
 CFHTTORUN = 'cal-53535-i-797722_1'
 
+
 class DiffimTestCases(unittest.TestCase):
```

Comment:
Ok. The problem seems to be that we have no way of checking that example code runs so we get bit rot. The memory leaks are generally easy by adding some `del` calls in the `tearDown`. Fixing the example code sounds like it's beyond the remit of this work. Talk to other UW people and work out whether a new ticket should be filed. I think we have a general issue with bit rot in examples. Ideally we would have a test target that runs all example code.